tvtk/tools/visual.py

```diff
@@ -1392,6 +1392,7 @@ class Box(HasTraits):
     polydata = Instance(tvtk.PolyData, ())
     actor = Instance(tvtk.Actor, ()) # tvtk Actor, for the usual pipeline architecture.
     property = Instance(tvtk.Property)
+    normals = Instance(tvtk.PolyDataNormals, ())
```

Comment:
It isn't clear why adding normals should change the position at all?  I am not able to understand the commit message and change.  What is the problem you are facing and why are you making this change?  The poly data normals should not affect the box as it will only smooth small angles and not larger angles like the one for a box.